src/libs/SidebarUtils.ts

```diff
@@ -105,6 +105,9 @@ function getOrderedReportIDs(
         if (!report) {
             return;
         }
+        if (report?.type === CONST.REPORT.TYPE.PAYCHECK || report?.type === CONST.REPORT.TYPE.BILL) {
```

Comment:
Given that we're referencing these in multiple places now. Maybe we should add something like `CONST.UNSUPPORTED_REPORT_TYPES` or something.